src/plugins/DistanceMeasurementsPlugin/DistanceMeasurement.js

```diff
@@ -1,8 +1,8 @@
-import {Dot3D} from "../lib/ui/index.js";
-import {Wire} from "../lib/html/Wire.js";
-import {Label} from "../lib/html/Label.js";
-import {math} from "../../viewer/scene/math/math.js";
-import {Component} from "../../viewer/scene/Component.js";
+import { Dot3D } from "../lib/ui/index.js";
```

Comment:
Let's not change code formatting in fix commits and branches, makes it really hard to follow what actually changed.